[2021-02-28 09:18:08,101] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: ga_to_postgres_dag.get_ga_data 2021-02-28T08:18:04.815680+00:00 [queued]>
[2021-02-28 09:18:08,106] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: ga_to_postgres_dag.get_ga_data 2021-02-28T08:18:04.815680+00:00 [queued]>
[2021-02-28 09:18:08,106] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 09:18:08,106] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-28 09:18:08,106] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 09:18:08,111] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): get_ga_data> on 2021-02-28T08:18:04.815680+00:00
[2021-02-28 09:18:08,113] {standard_task_runner.py:52} INFO - Started process 35259 to run task
[2021-02-28 09:18:08,119] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'ga_to_postgres_dag', 'get_ga_data', '2021-02-28T08:18:04.815680+00:00', '--job-id', '22', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ga_to_postgres_dag.py', '--cfg-path', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmplzzu0vet', '--error-file', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmpkosbnse9']
[2021-02-28 09:18:08,121] {standard_task_runner.py:77} INFO - Job 22: Subtask get_ga_data
[2021-02-28 09:18:08,148] {logging_mixin.py:104} INFO - Running <TaskInstance: ga_to_postgres_dag.get_ga_data 2021-02-28T08:18:04.815680+00:00 [running]> on host 120.0.168.192.in-addr.arpa
[2021-02-28 09:18:08,170] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ga_to_postgres_dag
AIRFLOW_CTX_TASK_ID=get_ga_data
AIRFLOW_CTX_EXECUTION_DATE=2021-02-28T08:18:04.815680+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-28T08:18:04.815680+00:00
[2021-02-28 09:18:08,171] {logging_mixin.py:104} INFO - get ga data
[2021-02-28 09:18:08,171] {python.py:118} INFO - Done. Returned value was: None
[2021-02-28 09:18:08,174] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=ga_to_postgres_dag, task_id=get_ga_data, execution_date=20210228T081804, start_date=20210228T081808, end_date=20210228T081808
[2021-02-28 09:18:08,186] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-28 09:18:08,216] {local_task_job.py:146} INFO - Task exited with return code 0
